Remotion/Web/UnitTests/Core/UI/Controls/CommandTests/CommandTestHelper.cs

```diff
@@ -249,10 +239,10 @@ public Command CreateNoneCommand (IWebSecurityAdapter webSecurityAdapter = null,
       return command;
     }
 
-    public Command CreateNoneCommandAsPartialMock ()
+    public Mock<Command> CreateNoneCommandAsPartialMock ()
     {
-      Command command = _mocks.PartialMock<Command> (CommandType.None,  (IWebSecurityAdapter) null, (IWxeSecurityAdapter) null);
-      InitializeNoneCommand (command);
+      var command = new Mock<Command> (CommandType.None, (IWebSecurityAdapter) null, (IWxeSecurityAdapter) null) { CallBase = true };
```

Comment:
There is no `.CallOriginalMethod (OriginalCallOptions.NoExpectation)` here, are you sure that `{ CallBase = true }` must be set?

Reply:
yes `{ CallBase = true }` must be set, because in `Rhino.Mocks` the mock was created as a `PartialMock`